drivers/wifi/nrf700x/zephyr/src/zephyr_fmac_main.c

```diff
@@ -313,19 +317,18 @@ enum wifi_nrf_status wifi_nrf_fmac_dev_add_zep(struct wifi_nrf_drv_priv_zep *drv
 	}
 
 	status = wifi_nrf_fmac_ver_get(rpu_ctx,
```

Comment:
Lets rename this to wifi_nrf_fmac_fw_ver_get

Reply:
Better to leave the FMAC API generic as we can always extend the args .